pkg/uninstall/uninstall.go

```diff
@@ -46,13 +50,14 @@ func (obj *uninstaller) UninstallBlocking(ctx context.Context, pkg ctrlpkg.Packa
 }
 
 // Uninstall deletes the v1alpha1.Package custom resource from the cluster.
-func (obj *uninstaller) Uninstall(ctx context.Context, pkg ctrlpkg.Package, isDryRun bool) error {
+func (obj *uninstaller) Uninstall(ctx context.Context, pkg ctrlpkg.Package, isDryRun bool, deleteNamespace bool) error {
```

Comment:
As discussed in a previous PR, this feature should only be available for `UninstallBlocking`, at least for now.